src/Moryx.ControlSystem/VisualInstructions/VisualInstructorExtensions.cs

```diff
@@ -77,20 +77,20 @@ public static long Execute(this IVisualInstructor instructor, string title, IVis
                 title,
                 parameter,
                 EnumInstructionResult.PossibleResults(typeof(T)),
-                result => { callback(EnumInstructionResult.ResultToEnumValue(typeof(T), result.Result)); });
+                result => callback(EnumInstructionResult.ResultToGenericEnumValue<T>(result.SelectedResult)));
```

Comment:
Maybe this is also an example to put into the migration guide. At least I couldn't make out on first glance why this change had to be done in the way you did it 🤔

Reply:
Because it did not compile, the old method still exists, but here an overload was called that did not match the caller signature.